src/Writers/LaTeXWriter.jl

```diff
@@ -43,9 +43,11 @@ const DOCUMENT_STRUCTURE = (
 )
 
 function render(doc::Documents.Document)
+    task_local_storage(:doc, doc)
     mktempdir() do path
         cd(path) do
             local file = replace("$(doc.user.sitename).tex", " ", "")
+            cp(joinpath(doc.user.root, doc.user.source, "assets"), "assets")
```

Comment:
These are static user assets, right? I.e. this is not related to _generated_ images per say?